src/components/Pane/Pane.Props.ts

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Pane } from './Pane';
+
+export interface IPaneProps extends React.Props<Pane> {
+    /**
+     * Whether the pane is displayed.
+     * @default false
+     */
+    hidden?: boolean;
+
+    /**
+     * Mode of the pane.
+     * @default PaneMode.push
+     */
+    paneMode?: PaneMode;
+
+    /**
+     * Has the close button visible.
+     * @default true
+     */
+    closeButtonHidden?: boolean;
```

Comment:
Thanks, I like the naming guidelines and showed them to my own team as well.  I welcome the improvements.